README.md

```diff
@@ -530,4 +531,33 @@ response = client.completion.create(
 )
 ```
 
+### Azure
+
+If you wish to interact with your Azure endpoint on Azure AI Studio, you can use the `AI21AzureClient`
+and `AsyncAI21AzureClient`.
+
+The following models are supported on Azure:
+
+- `jamba-instruct`
+
+```python
+from ai21 import AI21AzureClient
+from ai21.models.chat import ChatMessage
+
+client = AI21AzureClient(
+  base_url="https://your-endpoint.inference.ai.azure.com/v1/chat/completions",
+  api_key="<your api key>",
```

Comment:
maybe
base_url="https://<YOUR_ENDPOINT>.inference.ai.azure.com/v1/chat/completions",
api_key="<YOUR_API_KEY>",

A few questions:
1. Is it clear what "your endpoint" is, and where they can get it?
2. Is the API key the AI21 API key, from the playground, or a different key determined by Azure? It would be good to clarify that here.